shell/src/test/java/org/apache/accumulo/shell/ShellConfigTest.java

```diff
@@ -29,14 +29,14 @@
 import java.io.PrintStream;
 import java.nio.file.Files;
 
+import org.jline.reader.impl.LineReaderImpl;
```

Comment:
Is it safe to use the impl here? Doing so seems fragile.

Reply:
I forgot to update this test to use a dumb terminal like the other tests.